vertical-pod-autoscaler/e2e/v1/recommender.go

```diff
@@ -146,6 +148,46 @@ var _ = RecommenderE2eDescribe("Checkpoints", func() {
 	})
 })
 
+var _ = RecommenderE2eDescribe("CronJob", func() {
+	f := framework.NewDefaultFramework("vertical-pod-autoscaling")
+
+	ginkgo.It("recommendations are served", func() {
+		cronJob := framework.NewTestCronJob("hamster-cronjob", "*/3 * * * *", batchv1beta1.AllowConcurrent,
```

Comment:
My intention was for this to use the SetupHamsterCronJob here as well.

Reply:
Moved SetupHamsterCronJob from updater.go to common.go, so it can be reused in both updater.go and recommender.go